src/content/api/cli.md

```diff
@@ -70,49 +152,91 @@ If your project structure is as follows -
-	| bundle.js | 1.54 kB | 0 [emitted] | index       |
-	[0] ./src/index.js 51 bytes {0} [built]
-	[1] ./src/others.js 29 bytes {0} [built]
+Hash: 07a1c2d056e28ff1cab2
+Version: webpack 4.44.2
+Time: 174ms
+Built at: 10/10/2020 11:37:09 am
+  Asset       Size  Chunks             Chunk Names
+main.js  952 bytes       0  [emitted]  main
+Entrypoint main = main.js
+[0] ./src/index.js 30 bytes {0} [built]
+[1] ./src/others.js 0 bytes {0} [built]
 ```
 
 ```bash
-webpack ./src/index.js ./src.index2.js -o dist/bundle.js
+webpack ./src/index.js ./src/others2.js -o dist/
 ```
 
 This will form the bundle with both the files as separate entry points.
 
 ```bash
-	| Asset     | Size    | Chunks        | Chunk Names   |
-	|-----------|---------|---------------|---------------|
-	| bundle.js | 1.55 kB | 0,1 [emitted] | index, index2 |
-	[0] ./src/index.js 51 bytes {0} [built]
-	[0] ./src/index2.js 54 bytes {1} [built]
-	[1] ./src/others.js 29 bytes {0} {1} [built]
+Hash: fad168056241c7181505
+Version: webpack 4.44.2
+Time: 175ms
+Built at: 10/10/2020 11:38:28 am
+  Asset        Size  Chunks             Chunk Names
+main.js  1010 bytes       0  [emitted]  main
+Entrypoint main = main.js
+[0] multi ./src/index.js ./src/others2.js 40 bytes {0} [built]
+[1] ./src/index.js 30 bytes {0} [built]
+[2] ./src/others.js 0 bytes {0} [built]
+[3] ./src/others2.js 0 bytes {0} [built]
+```
+
+## Default Configurations
+
+CLI will look for some default configurations in the path of your project, here are the config files picked up by CLI.
+
+If no `mode` is supplied via flags or config then this is the lookup order in increasing order
+
+> example - config file lookup will be in order of .webpack/webpack.config.development.js > webpack.config.development.js > webpack.config.js
+
+```md
+'webpack.config',
+'webpack.config.dev',
+'webpack.config.development',
+'webpack.config.prod',
+'webpack.config.production',
+'.webpack/webpack.config',
+'.webpack/webpack.config.none',
+'.webpack/webpack.config.dev',
+'.webpack/webpack.config.development',
+'.webpack/webpack.config.prod',
+'.webpack/webpack.config.production',
+'.webpack/webpackfile',
 ```
 
+If `mode` is supplied, say `production` then config looking order will be -
 
-### Common Options
+`.webpack/webpack.config.production.* > .webpack/webpack.config.prod.* > webpack.config.production.* > webpack.config.prod.* > webpack.config.*`
+
+## Common Options
 
 W> Note that Command Line Interface has a higher precedence for the arguments you use it with than your configuration file. For instance, if you pass [`--mode="production"`](/configuration/mode/#usage) to webpack CLI and your configuration file uses `development`, `production` will be used.
 
-__List all of the options available on the cli__
+__List all of the commands and flags available on the cli__
 
 ```bash
 webpack --help
 webpack -h
 ```
 
+__Show help for a single command or flag__
+
+```bash
+webpack <command> --help
+webpack --<flag> --help
```

Comment:
```suggestion
webpack --help --<flag> 
```

Reply:
both are valid right?

Reply:
I dont think so. Can you once try Locally?

Reply:
![image](https://user-images.githubusercontent.com/21009455/95653469-a0ce6900-0b16-11eb-9a7d-3317d348be78.png)
 yep valid

Reply:
Interesting, I was trying with `--mode`. 

![Screenshot at 2020-10-10 16-37-43](https://user-images.githubusercontent.com/46647141/95653540-15a1a300-0b17-11eb-9321-3e58b94b643e.png)

Reply:
It works for some flags only. The behavior of `webpack --flag --help` is inconsistent. Hence I think we should document `webpack --help --flag`. 

![Screenshot at 2020-10-11 08-11-21](https://user-images.githubusercontent.com/46647141/95669047-77a2ec80-0b99-11eb-84a4-ae889924f940.png)

Reply:
yeah lets remove it